api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -159,36 +156,43 @@ ListRuntimeResponse toApiListRuntimeResponse(
 
   @Mapping(target = "createdDate", source = "auditInfo.createdDate")
   @Mapping(target = "toolDockerImage", source = "runtimeImages")
-  @Mapping(target = "configurationType", ignore = true)
-  @Mapping(target = "gceConfig", ignore = true)
-  @Mapping(target = "gceWithPdConfig", ignore = true)
-  @Mapping(target = "dataprocConfig", ignore = true)
   @Mapping(
       target = "googleProject",
       source = "cloudContext",
       qualifiedByName = "legacy_cloudContextToGoogleProject")
+  @Mapping(
+      target = "configurationType",
+      source = "labels",
+      qualifiedByName = "mapRuntimeConfigurationLabels")
+  // these 3 set by getRuntimeAfterMapper()
+  @Mapping(target = "gceConfig", ignore = true)
+  @Mapping(target = "gceWithPdConfig", ignore = true)
+  @Mapping(target = "dataprocConfig", ignore = true)
   Runtime toApiRuntime(LeonardoGetRuntimeResponse runtime);
 
   @Mapping(target = "createdDate", source = "auditInfo.createdDate")
+  @Mapping(
+      target = "googleProject",
+      source = "cloudContext",
+      qualifiedByName = "legacy_cloudContextToGoogleProject")
   @Mapping(target = "autopauseThreshold", ignore = true)
   @Mapping(target = "toolDockerImage", ignore = true)
-  @Mapping(target = "configurationType", ignore = true)
+  @Mapping(target = "errors", ignore = true)
+  @Mapping(
+      target = "configurationType",
+      source = "labels",
+      qualifiedByName = "mapRuntimeConfigurationLabels")
+  // these 3 set by listRuntimeAfterMapper()
   @Mapping(target = "gceConfig", ignore = true)
   @Mapping(target = "gceWithPdConfig", ignore = true)
   @Mapping(target = "dataprocConfig", ignore = true)
-  @Mapping(target = "errors", ignore = true)
-  @Mapping(
-      target = "googleProject",
-      source = "cloudContext",
-      qualifiedByName = "legacy_cloudContextToGoogleProject")
   Runtime toApiRuntime(LeonardoListRuntimeResponse runtime);
 
   RuntimeError toApiRuntimeError(LeonardoClusterError err);
 
   @AfterMapping
   default void getRuntimeAfterMapper(
       @MappingTarget Runtime runtime, LeonardoGetRuntimeResponse leonardoGetRuntimeResponse) {
-    mapRuntimeLabels(runtime, leonardoGetRuntimeResponse.getLabels());
```

Comment:
made this call more explicit - see the references to mapRuntimeConfigurationLabels